docker-compose.yml

```
@@ -30,15 +30,23 @@ services:
       - 'base-builder-data:/var/lib/ocurrent'
       - "capnp-secrets:/capnp-secrets"
   worker:
-    image: ocurrent/ocluster-worker:live
-    platform: linux/amd64
+    build:
+      dockerfile: docker/worker/Dockerfile
+      context: .
+    platform: linux
     command:
       - --connect=/capnp-secrets/pool-linux-x86_64.cap
+      # - --connect=/capnp-secrets/pool-linux-arm64.cap  # NOTE replace with arm64 if running on macos/arm64
       - --name=local
       - --allow-push=ocurrentbuilder/staging,ocurrent/opam-staging
       - --capacity=1
       - --state-dir=/var/lib/ocluster
+      - --store=rsync:/var/cache/obuilder
```

It only works if I make the following change:
```suggestion
      - --obuilder-store=rsync:/var/cache/obuilder
```

Is the `--store` argument working for you?

(I am running `git submodule sync --recursive && git submodule update --recursive` to get the right submodules.)